src/shmem.c

```diff
@@ -584,7 +589,7 @@ bool realloc_shm(SharedMemory *sharedMemory, const size_t size, const bool resiz
 		logg("FATAL: realloc_shm(): mremap(%p, %zu, %zu, MREMAP_MAYMOVE): Failed to reallocate \"%s\": %s",
 		     sharedMemory->ptr, sharedMemory->size, size, sharedMemory->name,
 		     strerror(errno));
-		exit(EXIT_FAILURE);
+		return false;
```

Comment:
I assume there doesn't need to be a change to the code that calls `realloc_shm` to expect returns instead of the function exiting on it's own?

Reply:
Thanks for your comment. There isn't really a proper way of handling these kinds of out-of-memory issues. I'll undo this to `exit()` because it's easier than ensuring down the road that everything is always properly cascading it downwards.